public_html/wp-content/plugins/wordcamp-speaker-feedback/includes/admin.php

```diff
@@ -206,6 +207,54 @@ function() {
 	}
 }
 
+/**
+ * Display the count of unapproved speaker feedback comments, injected into the admin menu.
+ *
+ * Run on current_screen so we can conditionally add this to the top-level CPT item, or the Feedback item in
+ * the submenu.
+ *
+ * @param WP_Screen $screen Current WP_Screen object.
+ */
+function add_feedback_bubble( $screen ) {
+	global $menu, $submenu;
+	if ( ! isset( $menu ) || empty( $menu ) ) {
+		return;
+	}
+	if ( ! current_user_can( 'moderate_' . COMMENT_TYPE ) ) {
+		return;
+	}
+
+	$pending_count = get_feedback( array(), array( 'hold' ), array( 'count' => true ) );
+	if ( $pending_count <= 0 ) {
+		return;
+	}
```

Comment:
It seems like it would be more consistent with the rest of the plugin to use `count_feedback` here instead...

Reply:
I don't think `count_feedback` returns unapproved comments, but maybe I read over it too quickly

Reply:
Oh, is `moderated` the count of unapproved/pending? 3 terms for the same thing? 🙃

Reply:
Yep, see previous comment about how much I love comments in Core 😄